styles.css

```diff
@@ -0,0 +1,157 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #eceef1;
+  height: 100%;
+  font-family: Arial, Helvetica, sans-serif;
+}
```

Comment:
- I thing the height property is causing the superposition of your content on each other. Make it auto or just remove it :nerd_face: .